### PR TITLE
feat(coins): add DUST

### PR DESCRIPTION
**Pull request by jspook16**
PR to add DUST to (atomicDex-Desktop)

### CODE REVIEW

File: assets/config/0.5.3-coins.json [hunk 1]
```diff
@@ -1866,6 +1866,32 @@
...
+      },
+      {
+        "url": "78.141.233.64:10001"
+	"ws_url": "78.141.233.64:9001"
+      },
+      {
+	"url": "192.248.166.207:10001"
+	"ws_url": "192.248.166.207:9001"
+      }
+    ],
```
Comment (Milerius) [inline assets/config/0.5.3-coins.json:1887]: Can you format please?

**Review by cipig — DISMISSED** [review]

File: assets/config/0.5.3-coins.json (hunk 1 above)
Comment (jspook16) [reply assets/config/0.5.3-coins.json]: Made formatting change updates

**Comment by smk762** [comment]
Thanks for the PR @jspook16, logo looks rad. Can you pls also add a hexcolor for it e.g. in https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/atomic_defi_design/Dex/Constants/Style.qml#L277 ?

**Comment by jspook16** [comment]
> Thanks for the PR @jspook16, logo looks rad. Can you pls also add a hexcolor for it e.g. in https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/atomic_defi_design/Dex/Constants/Style.qml#L277 ?

Do I add both the light color and the dark color or just one of them?

**Comment by jspook16** [comment]
> > Thanks for the PR @jspook16, logo looks rad. Can you pls also add a hexcolor for it e.g. in https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/atomic_defi_design/Dex/Constants/Style.qml#L277 ?
> 
> Do I add both the light color and the dark color or just one of them?

I've added the hexcolor to style.qml file

**Review by smk762 — APPROVED** [review]

**Pull request merged**